frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,223 @@
+      >
+        Pay the membership fee for a group
+      </A>
+      <A
+        href="/validate-rosters"
+        class="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white"
+      >
+        Validate your Team Roster for an event
+      </A>
+    </div>
+  );
+};
+
+const StaffActions = () => {
+  return (
+    <div class="w-90 text-sm font-medium text-gray-900 bg-white border border-gray-200 rounded-lg dark:bg-gray-700 dark:border-gray-600 dark:text-white">
+      <A
+        href="/players"
+        class="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white"
+      >
+        View Registered players
+      </A>
+      <A
+        href="/validate-rosters"
+        class="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white"
+      >
+        Validate Event Roster
+      </A>
+    </div>
+  );
+};
+
+const Dashboard = () => {
+  const [store, { setLoggedIn, setData }] = useStore();
+
+  createEffect(() => {
+    if (!store.loggedIn) {
+      const navigate = useNavigate();
+      navigate("/login", { replace: true });
+    }
+  });
+
+  onMount(() => {
+    if (!store?.data?.username) {
+      fetchUserData(setLoggedIn, setData);
+    }
+  });
+
+  createEffect(() => {
+    if (store?.data?.player || store?.data?.wards) {
+      initFlowbite();
+    }
+  });
+
+  return (
+    <div>
+      <h1 class="text-4xl font-bold mb-4 text-red-500">
+        Welcome {store?.data?.full_name || store?.data?.username}!
+      </h1>
+      <div
+        id="accordion-flush"
+        data-accordion="collapse"
+        data-active-classes="bg-white dark:bg-gray-900 text-gray-900 dark:text-white"
+        data-inactive-classes="text-gray-500 dark:text-gray-400"
+      >
+        <Show when={store.data.player}>
+          <h2 id="accordion-heading-player">
+            <button
+              type="button"
+              class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+              data-accordion-target="#accordion-body-player"
+              aria-expanded="false"
```

Comment:
Is there a reason you changed this from what was implemented in 2189b8b14eb74d59e896aeacf8393a819c966200. 
I implemented this to let see a "call to action" if their profile is incomplete. But, may be we can also show the status in the accordion header itself. What was your reasoning?

Reply:
Ahh I think something went wrong when I rebased from newer commits.. I didnt change anything in Dashboard except the file name change xD

Reply:
Lemme change to latest code and push just for Dashboard.js

Reply:
Are you going to fix the rebase? Or do you want me to do it? :)

Reply:
So I see that only Dashboard.js is not upto date. So I'm copy pasting from the main brach Home.js code here and pushing it. That'll work right?

Reply:
Yes, that should work! :ok_hand:

Reply:
Pushed it